application/src/main/java/be/jpendel/application/SignupUC.java

```diff
@@ -11,6 +11,8 @@ public SignupUC(MemberRepository memberRepository) {
         this.memberRepository = memberRepository;
     }
 
+    //TODO brrrr 'new Member' ipv een builder
+    // Volgens mijn begrip mag de applicatie laag niet een Member rechtreeks aanmaken ..waar is CreateMemberCommand ?
```

Comment:
het command zou binnen komen in de applicatie service he.
Dat het domein geen mooie factory of builder aanbied is nt zo proper maar geen ramp. In de code van IDDD maakt Vernon daar ook zijn Calendar en zo aan. Wat ik aartslelijk vind en niet zou doen. maar het zit wel verstopt. Uw applicatie service verstopt de new.